src/microbe_stage/FloatingChunk.cs

```diff
@@ -143,13 +183,16 @@ public void Init(ChunkConfiguration chunkType, string? modelPath)
         {
             // Capacity is set to 0 so that no compounds can be added
             // the normal way to the chunk
-            ContainedCompounds = new CompoundBag(0);
+            Compounds = new CompoundBag(0);
```

Comment:
As compounds is initialized anyway, I think this line should be removed and the comment moved up to where the CompoundBag constructor is called.